### ISSUE TITLE
getting blurred result for trained model

### ISSUE THREAD

**Issue by rohitpaul23** [lyc0929/OOTDiffusion-train#27]
Have finetuned the model using smaller test images for 1k epochs. The loss graph is not converging and when inferencing on final and intermediate saved models, it gives a similar result to that of the base-trained model

So to be sure, I have trained the model on base SD1.5, I used 10 images and trained it for 500 epochs and it seems it's not learning as the result I am getting from the trained model is blurry
have

what could be the problem here
![prob](https://github.com/lyc0929/OOTDiffusion-train/assets/34906547/a3168e4f-d6f3-4aa6-bb2c-066f3c551c9d)

**Comment by rohitpaul23** [comment]
When debugged, it seems the model is not training. I was trying to train the model in fp16 with deepspeed. It run ok with fp32
